Tasks/AzureAppServiceManage/task.json

```diff
@@ -254,6 +292,12 @@
         "InstallingSiteExtension": "Installing site extension '%s' in Azure App Service.",
         "FailedToFetchAppServiceState": "Failed to fetch App Service State for App Service: %s, Error Code: %s, Error Message: %s",
         "UnableToFetchAuthorityURL": "Unable to fetch authority url.",
-        "UnableToFetchActiveDirectory": "Unable to fetch active directory resource id."
+        "UnableToFetchActiveDirectory": "Unable to fetch active directory resource id.",
+        "UnableToGetAppInsightsResource": "Unable to get app insights resource with name : %s",
+        "UnableToIdentifyResourceGroupNameForAppInsightsResource": "Unable to identify resource group name for app insights resource : %s",
+        "ApplicationInsightsNotSupportedForLinuxApp": "Application insights resource is being supported for linux web app : %s",
+        "FailedToConfigureAlwaysOnProperty": "Failed to configure always on property.",
```

Comment:
Can you also call out that for this property might not be supported for few app service plans and user can upgrade the plan.

Reply:
Service plan will be mentioned in error message being returned by REST API